R/pkg/R/sparkR.R

```diff
@@ -81,6 +81,7 @@ sparkR.stop <- function() {
 #' @param sparkExecutorEnv Named list of environment variables to be used when launching executors.
 #' @param sparkJars Character string vector of jar files to pass to the worker nodes.
 #' @param sparkRLibDir The path where R is installed on the worker nodes.
+#' @param sparkPackages Character string vector of packages
```

Comment:
Could you add to the comment here that the should be packages from `spark-packages.org` ? Also it would be great to add an example to the `@examples` block below